Modules/ITS/src/runITSTracksRootFileReader.cxx

```diff
@@ -53,7 +53,7 @@ class ITSTracksRootFileReader : public o2::framework::Task
 
     mFileTracks = std::make_unique<TFile>(filenameTracks.c_str(), "READ");
     if (!mFileTracks->IsOpen()) {
-      LOG(error) << "ITSTracksRootFileReader::init. Cannot open file: " << filenameTracks.c_str();
+      ILOG(Error) << "ITSTracksRootFileReader::init. Cannot open file: " << filenameTracks.c_str();
```

Comment:
```suggestion
      ILOG(Error) << "ITSTracksRootFileReader::init. Cannot open file: " << filenameTracks.c_str() << ENDM;
```
Infologger needs a terminator, otherwise the log will be stuck in the buffer until ENDM is received elsewhere.

Reply:
Thanks! Fixed